src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -123,8 +123,6 @@ bool ngraph::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngrap
     manager.register_pass<ngraph::pass::WeightsDequantizeToFakeQuantize>();
 
     auto common_fusions = manager.register_pass<ngraph::pass::GraphRewrite>();
-    common_fusions->add_matcher<ngraph::pass::DepthToSpaceFusion>();
-    common_fusions->add_matcher<ngraph::pass::ShuffleChannelsFusion>(false);
```

Comment:
Ad 1. Done
Ad 2. It doesn't have an impact on the behavior because:
- MocTransformations on the C++ side are part of `CommonOptimizations` and behavior remains the same.
- On the Python binding side, `MocTransformations` are used with flag `use_shapes` as `False`, so the additional checks will be performed